Resources/config/commands.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="FOS\UserBundle\Command\ActivateUserCommand" class="FOS\UserBundle\Command\ActivateUserCommand">
+            <tag name="console.command" />
+        </service>
+        <service id="FOS\UserBundle\Command\ChangePasswordCommand" class="FOS\UserBundle\Command\ChangePasswordCommand">
```

Comment:
please don't use FQCN as service ids. We don't want to register these classes in the autowirable types